ffi/dotnet-idsdk/IoticsIdLib/src/AgentIdentity.cs

```diff
@@ -0,0 +1,9 @@
+namespace IOTICS;
+
+public class AgentIdentity(Identity factory, string seed, string keyName, string id, string did) : PartyIdentity(factory, seed, keyName, id, did)
+{
+    public void Recreate() 
+    {
+        Factory.RecreateAgentIdentity(Seed, KeyName, Id);
+    }
+}
```

Comment:
No new line

Reply:
where?

Reply:
At the end of the file. Github highlight it with a red flag
![image](https://github.com/Iotic-Labs/iotics-identity-go/assets/54267049/b05dc0a9-90a4-4612-a2d6-40ddeee6778e)